tasks/db_deploy/orca_sql.py

```diff
@@ -1,7 +1,7 @@
 """
 Name: orca_sql.py
 
-Description: All of the SQL used for creating and migrating the ORCA schema.
+Description: SQL used for creating the ORCA database.
```

Comment:
Not sure how best to resolve this, but this and create_db.py seem to have conflicting names/uses. Would it make sense to move the contents of this file into create_db.py?